### PR TITLE
RCAL-438 Add support for Reference File Quantities

### PR DESCRIPTION
**Pull request by PaulHuwe**
Resolves [JP-nnnn](https://jira.stsci.edu/browse/JP-nnnn)
Resolves [RCAL-438](https://jira.stsci.edu/browse/RCAL-438)

Closes #

This PR adds support for quantities within the dark class for Roman. Unit tested against both Roman & JWST.

**Checklist**
- [x] added entry in `CHANGES.rst` (either in `Bug Fixes` or `Changes to API`)
- [ ] updated relevant tests
- [ ] updated relevant documentation
- [ ] updated relevant milestone(s)
- [x] added relevant label(s)

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 General
 -------
 
--
+-
```
Comment (schlafly) [inline CHANGES.rst:7]: Errant white space?

File: src/stcal/dark_current/dark_class.py [hunk 2]
```diff
@@ -36,9 +36,13 @@ def __init__(self, dims=None, dark_model=None):
             meta data set based on the arrays in the dark_model.
         """
         if dark_model is not None:
-            self.data = dark_model.data
+            if isinstance(dark_model.data,u.Quantity):
```
Comment (schlafly) [inline src/stcal/dark_current/dark_class.py:39]: flake8 will likely complain about a missing comma.

**Review by schlafly — APPROVED** [review]
Looks good to me.  Some minor linting comments inline.

File: src/stcal/dark_current/dark_class.py (hunk 2 above)
Comment (PaulHuwe) [reply src/stcal/dark_current/dark_class.py]: Space added.

**Review by ddavis-stsci — APPROVED** [review]
LGTM

**Review by kmacdonald-stsci — APPROVED** [review]

**Comment by PaulHuwe** [comment]
@hbushouse @kmacdonald-stsci Can one of you merge this, please? (I lack merge permissions for STCAL.)

**Comment by hbushouse** [comment]
> @hbushouse @kmacdonald-stsci Can one of you merge this, please? (I lack merge permissions for STCAL.)

Done

**Pull request merged**